src/admin/engineHeader.php

```diff
@@ -8,17 +8,17 @@
 $localvars  = localvars::getInstance();
 $enginevars = enginevars::getInstance();
 
-recurseInsert("acl.php","php"); 
+recurseInsert("acl.php","php");
 recurseInsert("includes/vars.php","php");
 recurseInsert('includes/functions.php',"php");
 
 recurseInsert('engineIncludes.php');
 
-formBuilder::process();
-formBuilder::ajaxHandler();
+// formBuilder::process();
+// formBuilder::ajaxHandler();
```

Comment:
These 2 lines ar required. 

If you don't have the ajax handler you will have a number of issues on other parts of the admin interface.